roles/crawler/tasks/configure-databases.yml

```diff
@@ -51,16 +59,17 @@
     always_run: true
     changed_when: false
 
-  - name: Create the raw schema.
-    command: psql -c 'CREATE SCHEMA raw;'
-    when: "'raw' not in schemas.stdout"
-    register: raw_schema_result
+  - name: Create the raw and features schemata.
+    command: psql -c 'CREATE SCHEMA {{ item }};'
+    when: "'item' not in schemas.stdout"
```

Comment:
You need to use `{{ item }}` here. Unlike with your default vars or results registered earlier in the tasklist, which are populated to the local Python task execution environment, you do actually need to use interpolation for each item when looping over an iterable in your `with:` directives. Otherwise, this task looks for the literal 'item' in `schemas.stdout`, which will always return true, and then the task will run and fail on re-provisioning because it will try to re-create the schemas, returning a non-zero exit code.